opentelemetry/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 - Bump MSRV to 1.64 [#1203](https://github.com/open-telemetry/opentelemetry-rust/pull/1203)
+- `opentelemetry` crate now only carries the API types #1186
```

Comment:
add a note saying "opentelemetry_sdk" crate must be referenced instead, if relying on SDK features.